iOS/FVM/Framework/Logging/Log.swift

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+internal class Log {
```

Comment:
Do we have some configuration where you can set a level of logging?

Reply:
<img width="511" alt="Screen Shot 2019-03-12 at 08 57 17" src="https://user-images.githubusercontent.com/12657908/54183675-fc50f480-44a4-11e9-9a1d-f1e01c302843.png">
<img width="509" alt="Screen Shot 2019-03-12 at 08 57 35" src="https://user-images.githubusercontent.com/12657908/54183677-fc50f480-44a4-11e9-959d-c8fdaede14d8.png">

Reply:
We can add custom flags, I believe, but that needs some more research. For now, there's DEBUG flag

Reply:
cool